qiskit/transpiler/passes/mapping/stochastic_swap.py

```diff
@@ -410,11 +410,11 @@ def _layer_permutation(layer_partition, layout, qubit_subset,
         logger.debug("layer_permutation: failed!")
         return False, None, None, None
 
-    edgs = best_edges.edges()
+    edges = best_edges.edges()
     trivial_layout = Layout.generate_trivial_layout(canonical_register)
     for idx in range(best_edges.size//2):
         slice_circuit.apply_operation_back(
-            SwapGate(), [trivial_layout[edgs[2*idx]], trivial_layout[edgs[2*idx+1]]], [])
+            SwapGate(), [trivial_layout[edegs[2*idx]], trivial_layout[edgs[2*idx+1]]], [])
```

Comment:
```suggestion
            SwapGate(), [trivial_layout[edges[2*idx]], trivial_layout[edges[2*idx+1]]], [])
```

Reply:
Thank you for your comment. I have just addressed this.